pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/invertedindex/InvertedIndexHandler.java

```diff
@@ -84,7 +92,7 @@ private void createInvertedIndexForColumn(ColumnMetadata columnMetadata)
       if (_segmentWriter.hasIndexFor(column, ColumnIndexType.INVERTED_INDEX)) {
```

Comment:
Remove this check? Same for other handlers

Reply:
yeah, seems able to. will update those places.